### ISSUE TITLE
Duplicate Entry when restarting a task

### ISSUE THREAD

**Issue by brbkrbrns** [kimai/kimai#5106]
### Describe the issue

When a task is restarted, there are always two new entries created, with one measuring the time as usual and the other having a duration of 0 minutes and no description.

This is happening when restarting a task from the "Time Tracking -> My times" table. Also tested when restarting in the upper right menu icon, too.

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

2.23.0

### How do you run Kimai?

Docker

### Which PHP version are you using?

8.3.12

**Comment by kevinpapst** [comment]
Please retest in the demo, I cannot reproduce that.

**Comment by brbkrbrns** [comment]
Hey @kevinpapst,

thank you for your response and your really helpful tool.

This is a complete strange behavior. Retested it in the demo, then a fresh and clean dev environment, then locally with a db dump from prod, ... . And everything is working fine. So I removed all containers, volumes and images from prod and rebuild it. And the same behavior occurs again. Tried using a different subdomain and also clear and disable the browser and cloudflare caches. Nothing is working. But for the moment I'll close this as it seems this is not caused by the kimai version itself.
